dotnetv3/AutoScaling/scenarios/AutoScale_Basics/AutoScale_Basics/AutoScaleBasics.cs

```diff
@@ -0,0 +1,247 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+  Before running this SDK for .NET (v3) code example, set up your development
```

Comment:
SDK for .NET (v3) --> AWS SDK for .NET (v3)

Reply:
Fixed